dcm4ceph-core/src/main/java/org/open_ortho/dcm4ceph/core/Cephalogram.java

```diff
@@ -120,7 +129,7 @@ public void init() {
         super.init();
 
         // Load defaults first.
-        instanceProperties = FileUtils.loadProperties(this.getClass(),
+        instanceProperties = FileUtils.loadProperties(
```

Comment:
Agree, later properties are re-read in code. So it does not make sense to load at this point.
Ironically that was the code that was fixed in the first place :)

But let me please suggest another way, in commit [f5c2779bc00bc67d1e01ec9a69c7d2a83fa70be2](https://github.com/open-ortho/dcm4ceph/pull/6/commits/f5c2779bc00bc67d1e01ec9a69c7d2a83fa70be2)

The idea is foolproofness of program.
So even if user did not read the manual and did not provide .properties file he still will get output .dcm file.
The user will still get some results. And a warning.

Reply:
@tttomat19 , you suggestion is good, but it would facilitate the generation of incorrect or incomplete DICOM files. If a DOB is unknown, you should probably set it null, or not add it in the DICOM header at all, rather than just adding a false default value of 0000-01-01. 

I think the default properties file can be used more as an example properties file than a default properties file. i guess i should create a ticket for this, to fix these up. We could, for example, pull out all the defaults from the code, and put them in the properties files for the different kind of input images. (Which is sort of leaning toward your initial suggestion of using existing tools, and wrapping them with appropriate XML data.).

I like the cleaning up of the redundant file readability check,

Reply:
Clear, reading ceph_defaults removed